playwright-tests/tests/editor.spec.js

```diff
@@ -276,4 +276,84 @@ test.describe("?page=projects&tab=editor", () => {
       await expect(warningText).toBeVisible();
     });
   });
+
+  test.describe("Import project from NEAR Catalog", () => {
+    test.beforeEach(async ({ page }) => {
+      await page.goto(`/${ROOT_SRC}?page=projects&tab=catalogImport`);
+    });
+    test.use({
+      storageState: "playwright-tests/storage-states/wallet-connected.json",
+    });
+
+    // should have visible projects
+    test("should have visible projects", async ({ page }) => {
```

Comment:
Does this test belong in "editor?

Reply:
Now that I think about it, should it not? It's in the "flow"